### PR TITLE
Issue/19

### PR DESCRIPTION
**Pull request by shntnu**
#19

### CODE REVIEW

**Comment by shntnu** [comment]
Also handles #22

**Comment by shntnu** [comment]
Messed up the test - try again!

File: cytominer_database/munge.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import os
+import pandas as pd
+
+
+def munge(config, source, target=None):
+    """
+
+    :param config: Configuration file.
+    :param source: Directory containing subdirectories that contain an object CSV file.
+    :param target: Output directory. If not specified, then it is same as``source``.
```
Comment (mcquin) [inline cytominer_database/munge.py]: "as ``source``"

Comment (mcquin) [inline cytominer_database/munge.py]: Need space in "as ``source``"

File: cytominer_database/munge.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+"""
+
+import click
+import cytominer_database.utils
+import os
+import pandas as pd
+
+
+def munge(config, source, target=None):
+    """
```
Comment (mcquin) [inline cytominer_database/munge.py]: Can you include a description of what this function does to each CSV file it finds?

**Review by mcquin — APPROVED** [review]

File: cytominer_database/munge.py (hunk 2 above)
Comment (shntnu) [reply cytominer_database/munge.py]: done

**Review by mcquin — APPROVED** [review]

**Pull request merged**